README.md

```diff
@@ -72,13 +72,30 @@ pip install attrs typing
 
 ## Creating the `.pyi` files
 
+
+### GUI
 1. Add this directory to the `Script Directories` in the Ghidra Script Manager
 2. Refresh the script list
 3. Run `generate_ghidra_pyi.py` (will be located under `IDE Helpers`)
 4. When a directory-selection dialog appears, choose the directory you'd like to save the `.pyi` files in.
 
+### CLI
+
+```bash
+$GHIDRA_ROOT/support/analyzeHeadless /tmp tmp -scriptPath $(pwd) -preScript generate_ghidra_pyi.py ./
```

Comment:
This does not seem to work for me - it's missing a file to analyze.
Can you make sure the command here works?

Also, it would fail without the docs unzipped. Maybe add another command to unzip them first?

Reply:
What is the issue you get? As long as the docs are unzipped (which for now requires the Ghidra install in $GHIDRA_ROOT to be running and having unpacked the docs sadly) this works. The lack of a program should only lead to `WARN  Neither the -import parameter nor the -process parameter was specified; therefore, the specified prescripts and/or postscripts will be executed without any type of program context. (HeadlessAnalyzer)`

Reply:
This looks correct to me.
```
Headless Analyzer Usage: analyzeHeadless
           <project_location> <project_name>[/<folder_path>]
             | ghidra://<server>[:<port>]/<repository_name>[/<folder_path>]
           [[-import [<directory>|<file>]+] | [-process [<project_file>]]]
           [-preScript <ScriptName>]
           [-postScript <ScriptName>]
           [-scriptPath "<path1>[;<path2>...]"]
           [-propertiesPath "<path1>[;<path2>...]"]
           [-scriptlog <path to script log file>]
           [-log <path to log file>]
           [-overwrite]
           [-recursive]
           [-readOnly]
           [-deleteProject]
           [-noanalysis]
           [-processor <languageID>]
           [-cspec <compilerSpecID>]
           [-analysisTimeoutPerFile <timeout in seconds>]
           [-keystore <KeystorePath>]
           [-connect <userID>]
           [-p]
           [-commit ["<comment>"]]
           [-okToDelete]
           [-max-cpu <max cpu cores to use>]
           [-loader <desired loader name>]

Please refer to 'analyzeHeadlessREADME.html' for detailed usage examples and notes.
```